internal/gouging/gouging.go

```diff
@@ -0,0 +1,124 @@
+package gouging
+
+import (
+	"fmt"
+	"math"
+
+	"gitlab.com/NebulousLabs/errors"
+	rhpv2 "go.sia.tech/renterd/rhp/v2"
+	"go.sia.tech/siad/modules"
+	"go.sia.tech/siad/types"
+)
+
+var (
+	// ErrGougingDetected is returned when price gouging was detected in one of
+	// the fields of either the price table or the host's external settings.
+	ErrGougingDetected = errors.New("price gouging detected")
+
+	downloadOverdriveEstimatePct = .1
```

Comment:
I added it because Luke talks about overdrive as well without necessarily knowing the context of the term in `skyd`. So I figured overdrive was a very general term when it comes to downloading. I'll remove it though I was more or less expecting this comment.

Reply:
(I just felt we _have_ to account for some overdrive percentage at some point simply because there will be overdrive and we're setting `max` prices.. that's also why it's so pessimistic at 10%)